bazel/repository_locations.bzl

```diff
@@ -948,8 +965,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
         project_name = "WebAssembly for Proxies (C++ host implementation)",
         project_desc = "WebAssembly for Proxies (C++ host implementation)",
         project_url = "https://github.com/proxy-wasm/proxy-wasm-cpp-host",
-        version = "579189940ee48ebf1fb1d6539483506bee89f0b4",
-        sha256 = "eedcc5d0e73a715d9361eda39b21b178e077ab4d749d6bf2f030de81f668d6d3",
+        version = "55cdd8efd1264c7d53de01836b84d422f2d1d2df",
+        sha256 = "e4dc3952cad5b0f205d72a25c9066336390e7784b9be4bdc9c5e839610010eb3",
```

Comment:
Please update to the merged commit and make sure to update release date in UTC time (2021-05-07).